src/templates/base/2019/contributors.html

```diff
@@ -262,27 +271,37 @@ <h1 class="title title-lg">
   </div>
   <div class="contributors {{ ' '.join(config.teams.keys()) }}">
     <div class="filter-container">
-      <p>
+      <p aria-live="polite" aria-atomic="true">
         {{ self.filter_by_team() }}
       </p>
-        {% for id, team in config.teams.items() %}
-          <button class="{{ id }}" type="button">{{ localizedTeamNames[id] if localizedTeamNames[id]|length else team.name }}</button>
+      <ul>
+        {% for id, team in localizedTeamNames.items() | accentless_sort() %}
+        {% if id in config.teams %} <!-- note not all team translations may be in this years config -->
+        <li>
+          <button class="{{ id }}" type="button" aria-pressed="false" title="{{ self.filter_by() }} {{ localizedTeamNames[id] if localizedTeamNames[id]|length else team.name }}">
+            {{ localizedTeamNames[id] }}
+          </button>
+        </li>
+        {% endif %}
         {% endfor %}
+      </ul>
     </div>
 
-    <div class="contributor-grid">
-      <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/README.md" class="contributor-join" aria-labelledby="jointheteam">
-        <svg xmlns="http://www.w3.org/2000/svg" role="img" width="174" height="100" viewBox="0 0 203 117">
-          <title id="jointheteam">{{ self.join_the_team_title() }}</title>
-          <path d="M0 0l11 89 71 4.4L65.9 117l40.7-22.4L201 89l2-79z" />
-          <text x="50%" y="47%">
-            {{ self.join_the_team_text() }}
-          </text>
-        </svg>
-        <img src="/static/images/avatars/12.jpg" alt="" height="100" width="100" loading="lazy" />
-      </a>
+    <ul class="contributor-grid">
+      <li>
+        <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/README.md" class="contributor-join" aria-labelledby="jointheteam">
+          <svg xmlns="http://www.w3.org/2000/svg" role="img" width="174" height="100" viewBox="0 0 203 117">
+            <title id="jointheteam">{{ self.join_the_team_title() }}</title>
```

Comment:
I think this is a correct usage pattern, but we might want to check for a double-announce or mis-announce when embedded SVG is used to provide a label. I personally prefer the visually-hidden CSS technique, as it allows things like browser translation.

Reply:
Yeah this is based on usage pattern 8 from [Deque’s Accessible SVG post](https://www.deque.com/blog/creating-accessible-svgs/), which seems the best, even if it does cause a double announcement in at least one case. Do browsers not translate SVG components like the `<title>` tag here?

Reply:
As I understand it, anything passed through `aria-*` has the chance of not being picked up by browser/3rd party translation (like, I18n-style), as it's an attribute. I usually err on the side of avoiding it because of that.

Reply:
But it’s `aria-labelledby` not `aria-label` so points to an element for that very reason, so bit confused what you are saying? It’s an SVG element rather than an HTML element but I *think* that’s still OK.

Reply:
And it’s a `<title>` SVG element not a `title` attribute. Is that the cause of confusion because you didn’t notice that and thought it was the attribute?

Reply:
Nope, just involving the ARIA attribute in the chain. Again, I don't think it would be a blocking merging issue if the issue was still open, just a corner case usability consideration.